frontend/src/Components/SignupForm/SignupForm.css

```diff
@@ -1,64 +1 @@
-/* .container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  height: 100vh;
```

Comment:
Did we intend to remove this?

Reply:
Yes I did. I plan to improve the overall CSS of the application and that code is not going to be relevant but the file is important and will contain the css styling for the user sign up form